src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private final Random random = new Random();
+
+    public Color getRandomColor() {
+        return Color.values()[random.nextInt(Color.values().length)];
```

Comment:
Consider storing the result of `Color.values()` in a local variable before the loop. Calling `values()` inside the loop causes the array to be cloned every time, which can impact performance. Instead, you can do it once outside the loop and use the local variable inside the loop.